slack/chat.py

```diff
@@ -47,3 +47,21 @@ def post_message(channel, text, **kwargs):
         data[key] = FIELD_ENCODERS.get(key, default_encoder)(value)
 
     return slack.http_client.post('chat.postMessage', data)
+
+
+def update(channel, timestamp, text, **kwargs):
```

Comment:
minor: I think it's cleaner if `timestamp` comes after `text`.

Reply:
i think timestamp is unique key into the message so might be nice to take prescedent

Reply:
actually yeh I can see it's cleaner to have targetting attributes bundled over consistency. in that case let's update the order of `data = {`